tests/test_colormap.py

```diff
@@ -23,10 +23,17 @@
 import unittest
 import random
 import matplotlib.pyplot as plt
-import gdal
 import colorcet as cc
 from colorella.colormap import ColorMap
 import matplotlib.colors as col
+import warnings
+gdal_warning = 'No GDAL Installation found. Without gdal the following functions are not available: from_gdal, to_gdal'
+try:
+    from osgeo import gdal
+    GDAL_INSTALLED = True
+except:
+    GDAL_INSTALLED = False
+    raise warnings.warn(gdal_warning)
```

Comment:
This should only be raised when a gdal function is explicitly called. It is enough when you define it in the colormap class and catch the warning in the test class.